CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+### ⚠️ Breaking Changes from `4.0-beta.8`
+Renamed `Components.navigationTitleView` -> `Components.titleContainerView`
```

Comment:
when everything else on this PR is good to go I suggest adding instructions on how to upgrade from previous versions (if still applicable)

Reply:
On the PR Description?